assets/js/components/dashboard-splash/dashboard-splash-service.js

```diff
@@ -36,15 +37,18 @@ class DashboardSplashService extends Component {
 		const { image, title, content, link, linkText, opposite } = this.props;
 
 		return (
-			<div className={ `googlesitekit-splash-service ${ opposite ? 'googlesitekit-splash-service--opposite' : '' }` } style={ { backgroundImage: `url(${ imagePath }service-${ image }.jpg)` } }>
+			<div
+				className={ classnames( 'googlesitekit-splash-service', { 'googlesitekit-splash-service--opposite': opposite } ) }
+				style={ { backgroundImage: `url(${ imagePath }service-${ image }.jpg)` } }
+			>
 				<div className="mdc-layout-grid">
 					<div className="mdc-layout-grid__inner">
-						<div className={ `
+						<div className={ classnames( `
 							mdc-layout-grid__cell
 							mdc-layout-grid__cell--span-5-tablet
-							mdc-layout-grid__cell--span-7-desktop
-							${ opposite ? 'mdc-layout-grid__cell--start-3-tablet mdc-layout-grid__cell--start-5-desktop mdc-layout-grid__cell--offset-5-tablet' : '' }
-						` }>
+							mdc-layout-grid__cell--span-7-desktop`,
+						{ 'mdc-layout-grid__cell--start-3-tablet mdc-layout-grid__cell--start-5-desktop mdc-layout-grid__cell--offset-5-tablet': opposite }
```

Comment:
Ouch, this one is tough to read! I think it'd be best to list each classname with the same conditional. I know it's a bit redundant logic-wise, but it'll improve code readability a lot, and `classnames` is fast 😄